packages/client/src/generation/TSClient/common.ts

```diff
@@ -126,7 +126,9 @@ Prisma.NullTypes = {
 export const notSupportOnBrowser = (fnc: string, browser?: boolean) => {
   if (browser)
     return `() => {
-  throw new Error(\`${fnc} is unable to be run \${runtimeDescription}.
+  throw new Error(\`${fnc} was imported from @prisma/client/index-browser.
```

Comment:
I am a little bit confused about the new message.

Bundlers wold route `@prisma/client` imports to `@prisma/client/index-browser` automatically if they are building frontend build. So, from users standpoint, they already do what the error message suggesting and would probably be confused even more.